lighthouse-core/config/default-config.js

```diff
@@ -369,42 +369,42 @@ const defaultConfig = {
       description: str_(UIStrings.a11yCategoryDescription),
       manualDescription: str_(UIStrings.a11yCategoryManualDescription),
       auditRefs: [
+        {id: 'accesskeys', weight: 3, group: 'a11y-navigation'},
```

Comment:
@paulirish did we officially decide what our criteria for breaking changes was for v4 going forward? do we need to introduce this as `weight: 0` to start or are we doing that starting v5?

Reply:
> do we need to introduce this as `weight: 0` to start or are we doing that starting v5?

I think since this is not in the performance section it's less of a big deal (people might appreciate it, actually).

It was also a misunderstanding that disabled this audit, so we can treat the temporary `weight: 0` as a bug that we're now fixing? :D